packages/flutter_tools/test/integration.shard/debug_adapter/flutter_adapter_test.dart

```diff
@@ -584,6 +584,52 @@ The relevant error-causing widget was:
 
       await dap.client.terminate();
     });
+
+    group('can step', () {
+      late BasicProject project;
+      late String breakpointFilePath;
+      late int breakpointLine;
+      late String expectedPrintLibraryPath;
+      setUp(() async {
+        project = BasicProject();
+        await project.setUpIn(tempDir);
+
+        breakpointFilePath = globals.fs.path.join(project.dir.path, 'lib', 'main.dart');
+        breakpointLine = project.topLevelFunctionBreakpointLine;
+        expectedPrintLibraryPath = globals.fs.path.join('pkg', 'sky_engine', 'lib', 'core', 'print.dart');
+      });
```

Comment:
FYI: The style guide forbids use of `setUp` and friends[^1] (though I don't particularly care to enforce this until it becomes unwieldy in individual cases). 

[^1]: https://github.com/flutter/flutter/blob/master/docs/contributing/Style-guide-for-Flutter-repo.md#make-each-test-entirely-self-contained

Reply:
Ah, thanks for the note - I've inlined this and removed `setUp` :)